### PR TITLE
added preferred provider indicator to vet tec search results

### PR DESCRIPTION
**Pull request by hughes-dustin**
## Description
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19180

## Testing done
Developer tested 

## Screenshots
Screen Shot 2019-07-15 at 11 07 36 AM

## Acceptance criteria
- [ ]  If the institution is a preferred provider (as indicated by the WEAMS.csv Preferred Provider column), "Preferred provider" is displayed in the top right corner of the institution card.
     - There is a yellow star icon to the left of the text.
- [ ]  If the institution is not a preferred provider, there is no gap in the card where the indicator would have been displayed.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by zurbergram — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Comment by hughes-dustin** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/gi/utils/render.jsx [hunk 1]
```diff
@@ -21,3 +21,14 @@ export const renderCautionFlag = result => {
     </div>
   );
 };
+
+export const renderPreferredProviderFlag = result => {
+  const { preferredProvider } = result;
+  if (!preferredProvider) return null;
+  return (
+    <div className="preferred-flag">
+      <i className="fa fa-star color-gold" />
```
Comment (U-DON) [inline src/applications/gi/utils/render.jsx]: You might want to use this class: `vads-u-color--gold`. Here's a list of the color utility classes we have available from Formation: https://design.va.gov/utilities/color

File: src/applications/gi/sass/partials/_gi-search-page.scss [hunk 2]
```diff
@@ -147,5 +149,21 @@
...
+      border-radius: 3px 3px 3px 3px;
+      font-size: 0.9em;
+      font-weight: bold;
+      padding-top: 1em;
+      padding-bottom: .25em;
+      padding-right: 1.5em;
+      margin: 1em 0 0 1em;
+
+      i.fa {
+        color: $color-gold;
```
Comment (U-DON) [inline src/applications/gi/sass/partials/_gi-search-page.scss]: Might not need this if you're already specifying the color through the class, I think.

File: src/applications/gi/utils/render.jsx (hunk 1 above)
Comment (hughes-dustin) [reply src/applications/gi/utils/render.jsx]: Changes have been added.

File: src/applications/gi/sass/partials/_gi-search-page.scss (hunk 2 above)
Comment (hughes-dustin) [reply src/applications/gi/sass/partials/_gi-search-page.scss]: Removed unnecessary styling.

File: src/applications/gi/sass/partials/_gi-search-page.scss [hunk 3]
```diff
@@ -1,3 +1,5 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
```
Comment (U-DON) [inline src/applications/gi/sass/partials/_gi-search-page.scss]: Seems like this is no longer necessary to import.

**Review by U-DON — APPROVED** [review]
Just one more minor thing, but LGTM!

**Pull request merged**